ftw/referencewidget/profiles/default_plone5/registry.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<registry>
+    <records interface="ftw.referencewidget.interfaces.IReferenceSettings" />
+
+    <records prefix="plone.resources/ftw.referencewidget" interface='Products.CMFPlone.interfaces.IResourceRegistry'>
+        <value key="js">++resource++ftw.referencewidget/refwidget.js</value>
+        <value key="deps">jquery</value>
+        <value key="css">
+            <element>++resource++ftw.referencewidget/refwidget.css</element>
+        </value>
+    </records>
+
+    <records prefix="plone.resources/ftw.referencewidget-handlebar_js" interface='Products.CMFPlone.interfaces.IResourceRegistry'>
+        <value key="js">++resource++ftw.referencewidget/handlebars.js</value>
+        <value key="deps">jquery</value>
+    </records>
+
+    <records prefix="plone.bundles/ftw.referencewidget-resources" interface="Products.CMFPlone.interfaces.IBundleRegistry">
+        <value key="resources">
+            <element>ftw.referencewidget</element>
+            <element>ftw.referencewidget-handlebar_js</element>
+        </value>
+        <value key="enabled">True</value>
+        <value key="depends">plone</value>
+        <value key="compile">True</value>
```

Comment:
I'm sorry, I was wrong about how the ressource registry works. The documentation seems to be wrong.

I suggest these changes:

```diff
-        <value key="compile">True</value>
+        <value key="compile">False</value>
+        <value key="jscompilation">++plone++referencewidget/ftw-referencewidget-compiled.js</value>
+        <value key="csscompilation">++plone++referencewidgets/ftw-referencewidget-compiled.css</value>
```

Discussion: https://4teamwork.slack.com/archives/C0D69FKRD/p1574865319018000